README.md

```diff
@@ -168,65 +172,12 @@ This instance will have all the prerequisite software set up for you.
 - Download and install Maven v3.6.0: https://maven.apache.org/install.html
 - Make sure to test the installation using: ```mvn -version```
 
-### Getting the code
+### <a name="detailed-code"></a> Getting the code
 
 - Clone the repo using: ```git clone https://github.com/Elegant-Queueing/q-backend.git```
 - Make sure you're on ```master``` since it has the code from the latest release
 
-### Firebase
-NOTE: The Q team already has a dev database. To get access to it, please contact the team. To create your own database, use the following information.
-
-First, set up the database using the video: https://drive.google.com/file/d/1TMxiXX76JmJRJR1CP60grtf9WiX7aRxs/view
-
-Then, make changes to the code using the video: https://drive.google.com/file/d/1eiZPQq5OnAkD-kf628ZCYe6tTYlACvDB/view
+### <a name="detailed-firebase"></a> Firebase
+NOTE: The Q team already has a dev database. To get access to it, please contact the team. Once you have access to it, make changes to the code using the video: https://drive.google.com/file/d/1eiZPQq5OnAkD-kf628ZCYe6tTYlACvDB/view
```

Comment:
can add how to contact us. since you mentioned access is only given to 403 people you can mention slack